recipes/odil/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "odil" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lamyj/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5dcac1e415a84af737970c2db7cde329220080c5aaae933917757772067050c8
+  patches:
+    - windows_uuid.patch  # [win]
+    - windows_boost_system_and_python.patch  # [win]
+
+build:
+  number: 0
+  skip: True  # [py<=35 or (win and vc<14)]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - boost
+    # - dcmtk
```

Comment:
Can you add a comment as to why this is commented out or remove it please?

Reply:
Done in a5c81313b, thanks.